docs/scripting.rst

```diff
@@ -0,0 +1,142 @@
+.. _launching_hoc_and_python_scripts:
+
+Running Python and HOC scripts
+==============================
+
+This section describes the various ways that you can **run** Python and HOC
+scripts.
+You may also want to refer to the :ref:`Python <python_prog_ref>` and
+:ref:`HOC <hoc_prog_ref>` reference sections for documentation of how to write
+scripts using the two languages.
+
+HOC scripts are always run using one of the NEURON executables, such as
+``nrniv``, ``special``, ``neurondemo`` or ``neurongui``.
+Python scripts can also be run using these executables, but may additionally be
+run using the standard ``python`` and ``ipython`` commands.
+
+Files passed to the NEURON executables are identified based on their file
+extensions, so:
+
+.. code-block:: shell
+
+    nrniv script.hoc
+    nrniv script.py
+    nrniv -python script.hoc
+    nrniv -python script.py
+
+Will all work as expected.
+It is also possible to execute code from the commandline, for example:
+
+.. code-block:: shell
+
+    nrniv -c "hoc_statement"
+    nrniv -python "python_statement"
```

Comment:
I think something more analagous to the `nrniv -c` example would be:
```
python -c "python_statement"
```

By contrast `nrniv -python "python_statement"` seems to do nothing other than launch NEURON with a Python prompt (in particular it does not seem to run the command):

```
% nrniv -python "print('hi')"
INFO : Using neuron-nightly Package (Developer Version)
NEURON -- VERSION 9.0.dev-1316-gec1a06c3b HEAD (ec1a06c3b) 2023-03-16
Duke, Yale, and the BlueBrain Project -- Copyright 1984-2022
See http://neuron.yale.edu/neuron/credits

>>> 

```

Reply:
Thanks -- it was meant to say `nrniv -python -c "python_statement"`, which does run the code.
I'll fix that, and also add `python -c` and `python  script.py` to the examples.